mmcls/datasets/base_dataset.py

```diff
@@ -71,10 +71,10 @@ def get_cat_ids(self, idx):
             idx (int): Index of data.
 
         Returns:
-            int: Image category of specified index.
+            cat_ids (np.ndarray): Image category of specified index.
```

Comment:
Better to use `list[int]` to align with mmdetection.

Reply:
done